### PR TITLE
feat(linter): Respec linter

### PR DESCRIPTION
**Pull request by marcoscaceres**
Simple linter with one rule, that checks for privacy and security sections. 

@halindrome, wdyt?

### CODE REVIEW

File: js/w3c/linter.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+      .some(function(text) {
+        var privOrSecRegex = /(privacy|security)/igm;
+        var considerationsRegex = /(considerations)/igm;
+        return privOrSecRegex.test(text) && considerationsRegex.test(text);
+      });
+  }
+
+  return {
+    run: function(conf, doc, cb) {
+      if (!conf.lint) {
```
Comment (marcoscaceres) [inline js/w3c/linter.js]: I've disabled it by default, so could be "beta tested".

**Comment by halindrome** [comment]
I like the concept.  It should only be on things that are rec track. And maybe check for accessibility as well?  Are there other requirements from the pub rules checker we could easily vet for too?

**Comment by marcoscaceres** [comment]
>  It should only be on things that are rec track. 

Good point! 

> And maybe check for accessibility as well?

Sure. Looking forward to hearing suggestions on what to check. 

> Are there other requirements from the pub rules checker we could easily vet for too?

Checking for http urls in links might be an easy one.

**Comment by marcoscaceres** [comment]
Basically, any *URI prop in config should be https...

**Comment by marcoscaceres** [comment]
Ok, added check for "unofficial" to bail out. And added HTTP checker. 🎉

**Comment by marcoscaceres** [comment]
Here it is in action on the manifest spec: 

screenshot 2016-07-13 17 26 01

**Comment by halindrome** [comment]
I am going to raise this with the A11Y community to get some feedback.  If you would prefer to merge this and create a new issue for the A11Y linting, I am okay with that.

**Pull request merged**